extensions/http-curl/processors/InvokeHTTP.h

```diff
@@ -177,8 +177,20 @@ class InvokeHTTP : public core::Processor {
       .withDefaultValue(magic_enum::enum_name(invoke_http::InvalidHTTPHeaderFieldHandlingOption::transform))
       .withAllowedValues(magic_enum::enum_names<invoke_http::InvalidHTTPHeaderFieldHandlingOption>())
       .build();
+  EXTENSIONAPI static constexpr auto UploadBandwidthLimit = core::PropertyDefinitionBuilder<>::createProperty("Upload Bandwidth Limit")
+      .withDescription("Maximum data per second to send, use 0 for unlimited.")
+      .withPropertyType(core::StandardPropertyTypes::DATA_SIZE_TYPE)
+      .withDefaultValue("0 B")
+      .isRequired(true)
+      .build();
+  EXTENSIONAPI static constexpr auto DownloadBandwidthLimit = core::PropertyDefinitionBuilder<>::createProperty("Download Bandwidth Limit")
```

Comment:
I would use the world "speed" instead of "bandwidth".

If we want to be precise, the word "bandwidth" for general network transfer speeds is inaccurate: modern networks can transfer much more data even on narrower bands than older protocols. In everyday speech, in the context of internet speed, the two words are used as synonyms, but in this context, I think "speed" is clearer.

Reply:
Updated in 4f1691020c4bb6b7c39c131d1aa15b2d0f2ecedd

Reply:
Im a bit torn on this because the current problem is that the dimension of the property doesnt match its name. 
Ideally to a "speed" we could accept something like 100 KBps, but that would require a much larger change, maybe we could  rename it so its name at least matches the expected unit e.g. "Upload data limit per seconds"

What do you guys think? @lordgamez  @szaszm

Reply:
It's not ideal, but I don't think renaming to "upload data limit per second" would make it much better. Technically it's more fitting, but it's still confusing, and the name is less like what a user would look for.

I'd prefer to go with the current version, and if we introduce the necessary units to the ecosystem later, then we can extend the range of accepted values to include the proper unit.

A compromise could be naming it "Upload Speed Limit (/sec)".

Reply:
I like the "Upload Speed Limit (/sec)" naming

Reply:
I think the "per second" is a good idea, maybe we could name them "Data Upload Limit Per Second" and "Data Download Limit Per Second"?

Reply:
I'd prefer to keep referring to "download speed" and "upload speed" in the property name.

Another alternative approach would be to use a custom property type, with no corresponding C2/NiFi validator, and just parse the proper data rate units in onSchedule. I'd like that the most, but it requires a bit more additional code.

Reply:
If there are no objections, I suggest we keep the name, and change the format to e.g. "MB/s" meaning mebibytes/sec (to be consistent with data size units), with agent-side validation, and no specific validator in the C2 manifest.